internal/controllers/clusterctl/config.yaml

```diff
@@ -28,7 +28,7 @@ data:
       url:          "https://github.com/rancher-sandbox/cluster-api-provider-elemental/releases/v0.7.0/infrastructure-components.yaml"
       type:         "InfrastructureProvider"
     - name:         "docker"
-      url:          "https://github.com/kubernetes-sigs/cluster-api/releases/v1.7.7/infrastructure-components-development.yaml"
+      url:          "https://github.com/rancher-sandbox/cluster-api/releases/v1.7.7/infrastructure-components-development.yaml"
```

Comment:
I was asking same question in https://github.com/rancher/turtles/pull/741#discussion_r1757148786 basically, but since we will have automation soon, it probably makes sense to add Docker from the fork